### PR TITLE
fix(reset): Migrate reset to treat, remove Reset component

### PR DESCRIPTION
**Pull request by mattsjones**

### CODE REVIEW

File: lib/components/Box/Box.tsx [hunk 1]
```diff
@@ -76,77 +78,76 @@ export const Box = ({
...
-            atoms.flexDirectionDesktop,
-            flexDirection,
-          ),
-      )}
-      {...restProps}
-    />
-  );
+  const ResetBox = createElement(component, {
+    className: useClassNames(
+      useReset(component),
```
Comment (markdalgleish) [inline lib/components/Box/Box.tsx]: This is wonderful ❤️

File: lib/components/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// Import reset file for high document order
```
Comment (markdalgleish) [inline lib/components/index.ts]: Delete this.

File: lib/themes/theme.d.ts [hunk 3]
```diff
@@ -84,87 +84,6 @@ export type Color =
...
-  | 'menu'
-  | 'nav'
-  | 'output'
-  | 'ruby'
-  | 'section'
-  | 'summary'
-  | 'time'
-  | 'mark'
-  | 'audio'
-  | 'video';
```
Comment (markdalgleish) [inline lib/themes/theme.d.ts]: 👋👋👋

File: package.json [hunk 4]
```diff
@@ -66,9 +65,11 @@
   },
   "homepage": "https://github.com/seek-oss/braid-design-system#readme",
   "dependencies": {
+    "@types/classnames": "^2.2.7",
     "@types/lodash": "^4.14.123",
     "@types/react": "^16.8.8",
     "basekick": "^3.1.1",
+    "classnames": "^2.2.6",
```
Comment (markdalgleish) [inline package.json]: Too soon 😄

**Comment by michaeltaranto** [comment]
Closing for now, we'll come back to reset at the end.

**Pull request closed without merging**